crates/cli/src/cli/mod.rs

```diff
@@ -133,71 +133,71 @@ enum Command {
 
 #[derive(Parser, PartialEq, Clone, Debug)]
 pub struct StartSimnet {
-    /// Path to the manifest
+    /// Path to the runbook manifest, used to locate the root of the project (eg. surfpool start --manifest-file-path ./txtx.toml)
     #[arg(
         long = "manifest-file-path",
         short = 'm',
         default_value = "./Surfpool.toml"
     )]
     pub manifest_path: String,
-    /// Set the Simnet RPC port
+    /// Set the Simnet RPC port (eg. surfpool start --port 8080)
     #[arg(long = "port", short = 'p', default_value_t = DEFAULT_RPC_PORT)]
     pub simnet_port: u16,
     /// Set the Simnet WS port
     #[arg(long = "ws-port", short = 'w', default_value_t = DEFAULT_WS_PORT)]
     pub ws_port: u16,
-    /// Set the Simnet host address
+    /// Set the Simnet host address (eg. surfpool start --host 127.0.0.1)
     #[arg(long = "host", short = 'o', default_value = DEFAULT_NETWORK_HOST)]
     pub network_host: String,
-    /// Set the slot time
+    /// Set the slot time (eg. surfpool start --slot-time 400)
     #[arg(long = "slot-time", short = 't', default_value_t = DEFAULT_SLOT_TIME_MS)]
     pub slot_time: u64,
-    /// Set a datasource RPC URL (cannot be used with --network). Can also be set via SURFPOOL_DATASOURCE_RPC_URL.
+    /// Set a datasource RPC URL (cannot be used with --network). Can also be set via SURFPOOL_DATASOURCE_RPC_URL. (eg. surfpool start --rpc-url https://api.mainnet-beta.solana.com)
     #[arg(long = "rpc-url", short = 'u', conflicts_with = "network")]
     pub rpc_url: Option<String>,
-    /// Choose a predefined network (cannot be used with --rpc-url)
+    /// Choose a predefined network (cannot be used with --rpc-url) (eg. surfpool start --network mainnet)
     #[arg(long = "network", short = 'n', value_enum, conflicts_with = "rpc_url")]
     pub network: Option<NetworkType>,
-    /// Display streams of logs instead of terminal UI dashboard (default: false)
+    /// Display streams of logs instead of terminal UI dashboard (default: false) (eg. surfpool start --no-tui)
```

Comment:
In the current version, if I pull the help for `surfpool start --help`, I see:
```
      --no-studio
          Disable Studio (default: true)

      --studio-port <STUDIO_PORT>
          Set the Studio port

          [default: 18488]
```

Could we get rid of the all the `(default: x)` components from the comments strings and instead do:
```rust
    /// Display streams of logs instead of terminal UI dashboard (eg. surfpool start --no-tui)
    #[clap(long = "no-tui", default_value_t = true)]
    pub no_tui: bool,
```
so we leave `clap` displaying the default values ?